### PR TITLE
docs: added subscription recipe example

### PR DESCRIPTION
**Pull request by shahednasser**
- Added an example for the subscription recipe
- Fixes to the subscription recipe (overview)

Closes DOCS-798

### CODE REVIEW

File: www/apps/resources/app/recipes/subscriptions/examples/standard/page.mdx [hunk 1]
```diff
@@ -0,0 +1,2288 @@
...
+
+## Step 3: Run Migrations
+
+To create a table for the `Subscription` data model in the database, start by generating the migrations for the Subscription Module with the following command:
+
+```bash
+npx medusa migrations generate subscriptionModuleService
+```
+
+This generates a migration in the `src/modules/marketeplace/migrations` directory.
```
Comment (yosefserkez) [inline www/apps/resources/app/recipes/subscriptions/examples/standard/page.mdx]: This generates a migration in the `src/modules/subscription/migrations` directory.

(`marketeplace` is incorrect)

File: www/apps/resources/app/recipes/subscriptions/examples/standard/page.mdx [hunk 2]
```diff
@@ -0,0 +1,2288 @@
...
+      "metadata",
+    ],
+    variables: {
+      filters: {
+        id: [req.params.id],
+      },
+    },
+  })
+  
+  const { metadata } = await remoteQuery(query)
```
Comment (yosefserkez) [inline www/apps/resources/app/recipes/subscriptions/examples/standard/page.mdx]: remoteQuery returns an array here.

`const [{ metadata }] = await remoteQuery(query)`

Comment (shahednasser) [reply www/apps/resources/app/recipes/subscriptions/examples/standard/page.mdx]: Thanks for the feedback @4till2 !

**Review by olivermrbl — APPROVED** [review]
Nice work!

**Pull request merged**